bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/UL/thq_4f_ckm_LO_ctcvcp/thq_4f_ckm_LO_ctcvcp_customizecards.dat

```diff
@@ -0,0 +1,3 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card yukawa 6 -172.5
```

Comment:
Is it set to "-172.5" on purpose? Would give you different cross sections when setting it to +172.5 right?
And is higgs mass set to what?

Reply:
We are following the same configuration used in the full RunII card. The Higgs mass is set to 125 GeV, we made it explicit in the new version of the cards.

Reply:
I did a quick test and I think this gives different cross sections.
This should be checked I think

Reply:
Dear @sihyunjeon , can you please clarify what gives a different cross section with respect to what? The minus value for the yukawa coupling is desired and naturally yields a different cross section.

Unless you mean that setting explicitly the Higgs mass changes the cross section, to which I guess the answer is that it depends on what is the model default for the Higgs mass. In CMS production, we conventionally set it to 125 afaik, so I think @ataliercio did well in setting it explicitly to 125.

Reply:
@vischia @ataliercio 
I looked into the UFO model file and it looks like the default yukawa coupling for top quark is set to +173 (positive value). Now here you set it to -172.5 which is a negative value so my question was is this really intended as from the model card its yukawa are by default set to positive values ==> I just wanted to make sure whether this sign flipping was on purpose or not. But it seems like it does, right?